docs/deployment/feature_gates.md

```diff
@@ -153,3 +154,4 @@ A *General Availability* (GA) feature is also referred to as a *stable* feature.
 | ShootSARotation                            | `gardener-apiserver`, `gardenlet`                                | Enables the feature to trigger automated service account signing key rotation for shoot clusters. |
 | HAControlPlanes                            | `gardener-scheduler`, `gardenlet`                                | HAControlPlanes allows shoot control planes to be run in high availability mode. |
 | WorkerPoolKubernetesVersion                | `gardener-apiserver`                                             | Allows to overwrite the Kubernetes version used for shoot clusters per worker pool (see [this document](../usage/worker_pool_k8s_versions.md)) |
+| DefaultSeccompProfile                      | `gardenlet`                                                      | Enables the defaulting of the seccomp profile for Gardener managed workload in the seed to RuntimeDefault. |
```

Comment:
I guess it would be also useful to add docs for operators explaining that the feature gate aims to cover ManagedSeeds with K8s < 1.25 and for K8s >= 1.25 we plan to rely on the kubelet's `SeccompDefault` feature.